### ISSUE TITLE
MXNet Binary wheels don't contain certain includes

### ISSUE THREAD

**Issue by ChaiBapchya** [apache/incubator-mxnet#18430]
## Description
MXNet Binary wheels released as part of  don't contain
Following 4 files

https://github.com/apache/incubator-mxnet/blob/382279e0d5b88b2bd05e6c00105fe1e9eb2bc600/tools/pip/MANIFEST.in#L18-L21

are supposed to be included but aren't

## To Reproduce
Pull the latest mxnet wheel
```
wget https://repo.mxnet.io/dist/python/cu102/mxnet_cu102-1.6.0-py2.py3-none-manylinux1_x86_64.whl
```
Unzip it into a folder "unzipped_wheel"
```
unzip mxnet_cu102-1.6.0-py2.py3-none-manylinux1_x86_64.whl -d unzipped_wheel
```
Search for any of these files in the unzipped wheel
```
find unzipped_wheel/ -name "LICENSE"
```

## What have you tried to solve it?

This needs to be included in a directory.
setup.py doesn't throw error while trying to package mxnet possibly why this isn't caught

**Comment by yajiedesign** [comment]
windows includes "include" from 20200531

**Comment by ChaiBapchya** [comment]
Can you point me to the PR that fixed it? So we do that for linux binaries?

**Comment by leezu** [comment]
@ChaiBapchya, @yajiedesign manually maintains the windows binaries. But it would be good to merge the code into the public repo

**Comment by ChaiBapchya** [comment]
Yes. @yajiedesign if you could point me to what fixed it, I can do that for other builds as well.
Thanks.

**Comment by yajiedesign** [comment]
@leezu  I'm trying to merge to CD recently,At present, the main obstacle is that there is no CD windows machine available.
@ChaiBapchya, It's modification setup.py Include "include".But I don't know which CD now use  setup.py.is https://github.com/apache/incubator-mxnet/tree/master/python/setup.py ?

**Comment by yajiedesign** [comment]
@ChaiBapchya 
the key point code.but "python/setup.py" is many different to my setup.py.I'm not sure where to add it
```
for root, dirs, files in os.walk(os.path.join(CURRENT_DIR, 'include')):
    for file in files:
        package_data['mxnet'].append(os.path.abspath(os.path.join(root, file)))
```
